doc/admin/installation/docker_smallscale.rst

```diff
@@ -58,12 +58,17 @@ Database
 --------
 
 Next, we need a database and a database user. We can create these with any kind of database managing tool or directly on
-our database's shell, e.g. for MySQL::
-
-    $ mysql -u root -p
-    mysql> CREATE DATABASE pretix DEFAULT CHARACTER SET utf8mb4 DEFAULT COLLATE utf8mb4_unicode_ci;
-    mysql> GRANT ALL PRIVILEGES ON pretix.* TO pretix@'localhost' IDENTIFIED BY '*********';
-    mysql> FLUSH PRIVILEGES;
+our database's shell, e.g. for POSTGRESQL::
+
+    $ psql postgres -U root
```

Comment:
This seems like a convoluted way of creating a user and database – why are those "\du" lines included? Why do you need to sign out and then use postgres as the pretix user?

Either this:
```SQL
$ psql
postgres=# CREATE DATABASE yourdbname;
postgres=# CREATE USER youruser WITH ENCRYPTED PASSWORD 'yourpass';
postgres=# GRANT ALL PRIVILEGES ON DATABASE yourdbname TO youruser;
```
or even just this
```
sudo -u postgres -i
createuser -W <username>
createdb -O <username> <dbname>
```
should be sufficient, shouldn't it?